configs/fine-tune-llm_v2/aws.yaml

```diff
@@ -0,0 +1,14 @@
+head_node_type:
+  name: head
+  instance_type: m5.2xlarge
+worker_node_types: []
+
+auto_select_worker_config: true
+worker_node_types: []
+
+aws:
+  TagSpecifications:
+    - ResourceType: instance
+      Tags:
+        - Key: as-feature-multi-zone
+          Value: "true"
```

Comment:
Why does GCP need an attached disk but not AWS?

Also advanced configs will be rejected in OA soon...so we'll need to figure out an appropriate solution here if this workspace template requires 250GB disk.

Reply:
For some reason, the GCP provisioned instances had smaller disks and I ran into out of disk when running the same job on GCP

Reply:
@shomilj can we adjust the GCP default disk sizes to be the same for AWS and GCP? Then we can get rid of this config.

Reply:
The defaults should be the same; investigating.

Reply:
they are the same (150 GB) -- can you link me to a job @shawnanyscale?

Reply:
ack. Updating the PR for now and will send links after